README.md

```diff
@@ -32,38 +32,45 @@ There are 2 analyses using Pseudomonas dataset in the `Pseudomonas` directory an
 | [Human_experiment_lvl_sim](Human/Human_experiment_lvl_sim.ipynb) | Human experiment-level gene expression simulation|
 
 
-## Computational Environment
+## How to run notebooks from simulate-expression-compendia
 
-All processing and analysis scripts were performed using the conda environment specified in `environment.yml`.
-To build and activate this environment run:
+In order to run this simulation on your own gene expression data the following steps should be performed:
 
+First you need to set up your local repository: 
+1. Clone the `simulate-expression-compendia` repository
+2. Set up conda environment by running the following command in the terminal:
 ```bash
 # conda version 4.6.12
 conda env create -f environment.yml
 
 conda activate simulate_expression_compendia
-```
 
-Install local modules into this environment:
-```
-pip install .
+pip install -e .
 ```
+3. Navigate to either the `Pseudomonas` or `Human` directories and run the notebooks.
 
-Note: R scripts were run using R 3.6.3 
 
-## How to run this simulation using your own data
+## How to run using your own data
 
 In order to run this simulation on your own gene expression data the following steps should be performed:
 
-First you need to set up your local repository: 
+First you need to set up your local repository and environment: 
 1. Clone the `simulate-expression-compendia` repository
-2. Set up conda environment using the command above
+2. Set up conda environment by running the following command in the terminal:
+```bash
+# conda version 4.6.12
+conda env create -f environment.yml
+
+conda activate simulate_expression_compendia
+
+pip install -e .
+```
 3. Create a new analysis folder in the main directory. This is equivalent to the `Pseudomonas` directory
-4. Copy `Pseudomonas_sample_lvl_sim.ipynb` and `Pseudomonas_experiment_lvl_sim.ipynb` into your analysis folder. 
+4. Copy `Pseudomonas_sample_lvl_sim.ipynb` or `Pseudomonas_experiment_lvl_sim.ipynb` into your analysis folder depending on if you would like to use the sample level or experiment level simulation approach. 
```

Comment:
You may want to add a sentence or link to what the sample level and experiment level simulation approaches are

Reply:
Thats a great idea. I have added a linkout to ponyo which has text in the functions to explain the method in more detail